package.json

```diff
@@ -37,6 +37,7 @@
         "lint:js": "eslint --report-unused-disable-directives --cache --ignore-path .gitignore --ext .js,.ts,.tsx ./",
         "lint": "yarn lerna run lint:styles && yarn lint:js",
         "lint-staged": "npx lint-staged",
+        "lint:shellcheck": "./scripts/shellcheck.sh",
```

Comment:
shouldn't we call this in some CI flow? otherwise it's not too much useful

Reply:
yup, another PR with github action is incoming (we talked about this on slack only)

Reply:
as of now there are still plenty of issues in shell scripts, I'm submitting PRs to solve all those
when all are solved it makes sense to block a PR with a github action (but not before everything is fixed imho)

Reply:
implemented in #6005